template/tests/SERVICE_NAME.service.spec.ts

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
This test file does not result in 100% test coverage. Please add appropriate tests.